lib/python/vdm/server/HTTPListener.py

```diff
@@ -846,7 +846,9 @@ def post(database_id):
 
         sync_configuration()
         Configuration.write_configuration_file()
-        return jsonify({'server': server, 'status': 1,
+        url = 'http://%s:%u/api/1.0/databases/%u/servers/%u/' % \
+                                  (__IP__, __PORT__, database_id, server_id)
+        return jsonify({'status': 201, 'statusString': 'OK', 'location': url, 'server': server,
```

Comment:
This will make location another property in the json response payload, which is not what we want to do. We want location url to be header in the flask response. 
resp.headers['Location'] = url